Tests/SwiftAlgorithmsTests/RandomSampleTests.swift

```diff
@@ -16,7 +16,7 @@ func validateRandomSamples<S: Sequence>(
   _ samples: [Int: Int],
   elements: S,
   expectedValue: Int,
-  file: StaticString = #file, line: UInt = #line
+  file: StaticString = #filePath, line: UInt = #line
```

Comment:
Could we resolve these warnings in this way instead? The `#filePath` attribute was introduced in Swift 5.8, but we're going to support back to Swift 5.7 in the next release of this package.

```suggestion
  file: StaticString = (#file), line: UInt = #line
```

Reply:
Thank you for your review. I have updated my code based on your suggestion.